content/sv/who.md

```diff
@@ -0,0 +1,4 @@
+Vem bör läsa detta dokument?
+==============================
+
+Alla utvecklare som utvecklar applikationer som körs som tjänst. DevOps och systemansvariga som produktionssätter eller hanterar sådana applikationer.
```

Comment:
```suggestion
Alla utvecklare som utvecklar applikationer som körs som tjänst (Software as a Service). DevOps och systemansvariga som produktionssätter eller hanterar sådana applikationer.
```

"körs som tjänst" does not directly associate with Software as a Service for me (even though it is correct), could clarify with my suggestion?